tests/hazmat/primitives/test_pkcs12.py

```diff
@@ -137,3 +139,138 @@ def test_buffer_protocol(self, backend):
         assert parsed_key is not None
         assert parsed_cert is not None
         assert parsed_more_certs == []
+
+    @pytest.fixture(scope='function')
```

Comment:
As I said on the other PR, I'm not wild about this fixtures pattern.

Even ignoring that, cert vs. cert2 is terrible naming.

Reply:
Removed